src/index.ts

```diff
@@ -146,10 +188,13 @@ const plugin: JupyterFrontEndPlugin<IAICellTracker> = {
       app.commands,
       notebookTracker,
       eventListener,
-      cellFooterTracker
+      cellFooterTracker,
+      magicProvider
     );
     return aiCellTracker;
   }
 };
 
-export default [plugin, agentCommands];
+export default [plugin, agentCommands, magicProviderPlugin];
+
+export * from './provider';
```

Comment:
Maybe at some point this repo will host multiple packages providing different functionalities. And this token could then be moved to a more appropriate place.